server/serializers/subscription_filters.go

```diff
@@ -0,0 +1,19 @@
+package serializers
```

Comment:
is the file name correct here?

Reply:
yes, I think so, do you have any other thoughts?

Reply:
I think filters don't fit well here. 
git_subscriptions or repo_subscriptions maybe?